api/src/test/java/org/pmiops/workbench/api/AdminAuthorityTest.java

```diff
@@ -88,9 +88,16 @@ public class AdminAuthorityTest {
    */
   @Test
   public void testAdminEndpointsHaveAuthority() throws Exception {
-    List<Class<?>> controllers = findClassesInApiPackageWithAnnotation(RestController.class);
+    List<Class<?>> controllers =
+        findClassesInApiPackageWithAnnotation(RestController.class).stream()
+            .filter(
+                c ->
+                    Arrays.stream(c.getInterfaces())
+                        .map(i -> i.getName())
+                        .anyMatch(n -> n.endsWith("ApiDelegate")))
+            .collect(Collectors.toList());
     Map<String, Class<?>> generatedApis =
-        findClassesInApiPackageWithAnnotation(Api.class).stream()
+        findClassesInApiPackageWithAnnotation(Validated.class).stream()
```

Comment:
I believe that @yonghaoy and I worked on this one together. Swagger 3 no longer makes use of the @API annotation, so this test needed to be updated to account for that change. More information about the annotation changes can be found here:
https://support.intershop.com/kb/index.php/Display/2914L4